cpp/ql/src/semmle/code/cpp/ir/implementation/raw/internal/TranslatedCall.qll

```diff
@@ -7,6 +7,7 @@ private import InstructionTag
 private import TranslatedElement
 private import TranslatedExpr
 private import TranslatedFunction
+private import semmle.code.cpp.models.implementations.Allocation
```

Comment:
Ideally we should not be importing from the `implementations` directory but from the `interfaces` directory, and then the interface needs to be expressive enough for what callers need. `Allocation` is a bit special because it's ported from an existing library that didn't use to have this split, but I hope we can make it work anyway.

Reply:
I've removed the need for this import now.